core/server/data/migrations/versions/3.22/01-removed-legacy-values-from-settings-table.js

```diff
@@ -0,0 +1,65 @@
+const ObjectId = require('bson-objectid');
+const logging = require('../../../../../shared/logging');
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up(options) {
+        const settingsKeys = ['force_i18n', 'permalinks', 'members_session_secret'];
+
+        logging.info(`Removing ${settingsKeys.concat(', ')} from "settings" table.`);
```

Comment:
@naz Think this is intended to be `.join` here instead of `.concat`, as `.join` will create a new string joining the elements in array, while `.concat` is to merge 2 arrays, and will just add an extra comma in the end leading to incorrect log line :)

Reply:
Thanks! Valid point, all fixed now ;)